core/src/test/java/org/apache/accumulo/core/util/FastFormatTest.java

```diff
@@ -117,4 +118,30 @@ public void testArrayOutOfBounds() {
     assertThrows(ArrayIndexOutOfBoundsException.class,
         () -> FastFormat.toZeroPaddedString(str, 4, 64L, 4, 16, new byte[] {'P'}));
   }
+
+  @Test
+  public void testHexString() {
```

Comment:
Would it be of benefit to add some assertions that this behaves the same as `String.format`? 
e.g.
`assertEquals(String.format("%016x", 64L), FastFormat.toHexString(64L));`